exercises/protein-translation/canonical-data.json

```diff
@@ -0,0 +1,143 @@
+{
+  "exercise": "protein-translation",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Translate input RNA sequences into proteins",
+      "comments": [
+        "Returns the name of the protein if given RNA is valid, "
+      , "else throws an error. "
+      ],
+      "cases": [
+        {
+          "description": "Methionine RNA sequence is identified",
```

Comment:
I think the " is identified" postfix can be removed, as it doesn't add much extra information I think. But maybe someone else disagrees?

Reply:
That makes sense to me - concise is good!